lib/errors.js

```diff
@@ -101,8 +114,15 @@ class CliError extends Error {
  * @extends CliError
  */
 class ProcessError extends CliError {
+    constructor(options) {
+        options = options || {};
+
+        options.message = options.message || `Error occurred running command: '${options.cmd}'`;
+        super(options);
+    }
+
     toString(verbose) {
-        let output = chalk.red(`Error occurred running command: '${this.options.cmd}'\n\n`);
+        let output = super.toStringBasics();
```

Comment:
can just use `this.toStringBasics` here b/c the method is inherited.

Reply:
yeah true 👍